doc/src/build/sui-local-network.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Create a local Sui network
+---
+
+Learn how to create a Sui network in your local environment. Use the [Sui Client CLI](cli-client.md) to interact with the local network.
+
+## Install Sui
+
+To create a local Sui network, first install Sui. See [Install Sui to Build](install.md).
+
+## Genesis
+
+To create the configuration files and objects for a local Sui network, run the `genesis` command. Genesis creates the network configuration files in the ~/.sui/sui_config folder. This includes a YAML file for fullnode, network, client, and each validator. It also creates a sui.keystore that stores client key pairs. 
+
+The network that genesis creates includes four validators and five user accounts that contain five coin objects each.
+
+   ```shell
+   $ sui genesis
+   ```
+
+The first time you use the client CLI, it creates a client.yaml file. If you use the default values, it connects to a Sui Devnet Full node. When you run genesis to create a local network, if the .sui/sui_config folder contains a client.yaml file, the genesis process warns you that the folder must be empty. You can use the `--force` argument to replace the configuration files, or use `--working-dir` to specify a different directory for the network configuration files.
+
+Use the following command to overwrite existing configuration files with default values.
+   ```shell
+   $ sui genesis --force
+   ```
+
+Use the following command to use a different directory to store the configuration files.
+```shell
+$ sui genesis --working-dir /workspace/config-files
+```
+
+The directory must already exist, and be empty, before you run the command.
+
+#### Embedded gateway
+
+You can use an embedded gateway with your local network. The gateway.yaml file contains information about the embedded gateway.
```

Comment:
let's note here that embedded gateway will soon be going away too

Reply:
added.